public/app/plugins/panel/table/renderer.ts

```diff
@@ -58,6 +58,18 @@ export class TableRenderer {
     if (!style.thresholds || !style.colors) {
       return null;
     }
+
+    if (style.thresholds.length >= style.colors.length) {
+      console.error(
+        // tslint:disable-next-line:max-line-length
+        `There must be max ${style.colors.length - 1} thresholds for ${style.colors.length} colors. You have ${
```

Comment:
should this be an error (thrown)?

Reply:
and be more descriptive to what the threshold format should be and means, lower / upper bound

Reply:
Currently the table does not render for me in v6.3.x and there's no info why. So I was thinking I should give user feedback what's wrong, did not found better way tho.

Reply:
The threshold format is already described in threshold input, no?

Reply:
> Currently the table does not render for me in v6.3.x, and there's no info why

That is why I think this needs to throw an error so it becomes visible, or have validation in the editor UI. console logging the error does not help that much,

Reply:
Got it. Is there any easy way to trigger AlertError popup from within that function?

Reply:
throw an exception should work

Reply:
I tried to check for exceptions in typescript files but there are none. Found only errors.

When I `throw new Error` (see last commit), it shows in console but it's not caught in UI.